### ISSUE TITLE
Ignoring unrecognized command: b'IN'

### ISSUE THREAD

**Issue by droberin** [mik3y/pymidi#9]
I get this every few seconds

```
WARNING:pymidi.ControlProtocol:Ignoring unrecognized command: b'IN'
WARNING:pymidi.ControlProtocol:Ignoring unrecognized command: b'BY'

```
Setting verbose mode crashes server:

```
Traceback (most recent call last):
  File "pymidi/server.py", line 157, in <module>
    server.serve_forever()
  File "pymidi/server.py", line 117, in serve_forever
    proto.handle_message(buffer, addr)
  File "/home/fakeusername/myvenv/lib/python3.6/site-packages/pymidi-0.3.0-py3.6.egg/pymidi/protocol.py", line 70, in handle_message
    self.logger.debug('rx: {}'.format(data.encode('hex')))
AttributeError: 'bytes' object has no attribute 'encode'

```

I'm connecting rtpMIDI tool by Tobias Erichsen from a Windows machine
also TouchDaw from an Android device. Both cause same errors.

**Comment by mik3y** [comment]
Thanks for the report! Both of these look like Python 3 support issues; treating `bytes` objects as strings. Will fix.

**Comment by droberin** [comment]
I'm planning to use this software in another software I want to release, so the best this gets the best the other can be. I'll let you know if I finally go for it.

Thanks for the incoming fix!

**Comment by mik3y** [comment]
No problem-o!

You can find a work-in-progress in the branch `mikey/py3`, which I just pushed. It fixes at least this python 3 issue, though there may be others. Caution - I will be pushing more changes and tests to this branch before merging, and I may rebase it; so feel free to check it out if you can tolerate sudden changes, otherwise just wait for me to close this.

**Comment by mik3y** [comment]
Should be resolved in #10 / v0.4.0 (now released). Please open a new issue if any further problems!
